src/workflows-app/components/SubmissionDetailsInputsTable.ts

```diff
@@ -0,0 +1,125 @@
+import { ReactNode, useState } from 'react';
+import { div, h } from 'react-hyperscript-helpers';
+import { AutoSizer } from 'react-virtualized';
+import { FlexTable, Sortable, tableHeight, TextCell } from 'src/components/table';
+import { getInputTableData, InputTableData } from 'src/workflows-app/utils/submission-utils';
+
+import { InputDefinition, RecordLookupInputSource } from '../models/submission-models';
+
+const rowWidth = 100;
+const rowHeight = 50;
+
+type SubmissionDetailsInputsTableProps = {
+  configuredInputDefinition: InputDefinition[];
+};
+
+const SubmissionDetailsInputsTable = ({ configuredInputDefinition }: SubmissionDetailsInputsTableProps): ReactNode => {
+  const [sort, setSort] = useState<{
+    field: 'taskName' | 'variable';
+    direction: 'asc' | 'desc';
+  }>({
+    field: 'taskName',
+    direction: 'asc',
+  });
+
+  const includeOptionalInputs = true;
+  const searchFilter = '';
+
+  const inputTableData: InputTableData[] = getInputTableData(
+    configuredInputDefinition,
+    searchFilter,
+    includeOptionalInputs,
+    sort
+  );
+
+  return div(
+    {
```

Comment:
Since the input + output tables use the same CSS and since we want them to be visually coherent, it might make sense to define this `style` once (perhaps in`submission-utils`, or a new file like `submission-tables-styles.ts`?) and reference it in both places.